release.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+tag="$1"
+sha="$2"
+
+echo "Creating a new release"
+echo "Tag  $tag"
+echo "SHA $sha"
+
+echo "Setting user name and email"
+git config --global user.name "a11y-insights"
+git config --global user.email "a11y-insights-team@microsoft.com"
+
+echo "git pull"
+git pull
+
+echo "Create a new branch"
+git checkout -b releases/$tag $sha
```

Comment:
@dbjorge suggested that we use a slightly longer name for the branch:
```suggestion
git checkout -b releases/gh/$tag $sha
```